manager/controllers/baseline_update_test.go

```diff
@@ -0,0 +1,105 @@
+package controllers
+
+import (
+	"app/base/core"
+	"app/base/database"
+	"app/base/models"
+	"app/base/utils"
+	"bytes"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUpdateBaseline(t *testing.T) {
+	utils.SkipWithoutDB(t)
+
+	core.SetupTestEnvironment()
+	baselineID := database.CreateBaseline(t)
+
+	data := `{
+		"name": "updated_name",
+		"inventory_ids": ["00000000-0000-0000-0000-000000000005", "00000000-0000-0000-0000-000000000006"],
+		"config": {
+			"to_time": "2021-01-01 12:00:00-04"
+		}
+	}`
+
+	w := httptest.NewRecorder()
+	path := fmt.Sprintf(`/%v`, baselineID)
+	req, _ := http.NewRequest("POST", path, bytes.NewBufferString(data))
+	core.InitRouterWithParams(BaselineUpdateHandler, 1, "POST", "/:baseline_id").ServeHTTP(w, req)
+
+	assert.Equal(t, http.StatusOK, w.Code)
+
+	var baseline models.Baseline
+	assert.Nil(t, database.Db.Model(&baseline).Where("id = ?", baselineID).Find(&baseline).Error)
+	assert.Equal(t, baseline.ID, baselineID)
+	assert.Equal(t, baseline.Name, "updated_name")
+
+	output := testUpdatedBaselineSystems(t, baselineID)
+
+	assert.Equal(t, 2, len(output.Data))
+	assert.Equal(t, "baseline_systems", output.Data[0].Type)
+	assert.Equal(t, "00000000-0000-0000-0000-000000000006", output.Data[0].ID)
+	assert.Equal(t, "00000000-0000-0000-0000-000000000005", output.Data[1].ID)
+
+	database.DeleteBaseline(t, baselineID, []string{
+		"00000000-0000-0000-0000-000000000005",
+		"00000000-0000-0000-0000-000000000006",
+	})
+}
+
+func TestUpdateBaselineWithEmptyFields(t *testing.T) {
+	utils.SkipWithoutDB(t)
+
+	core.SetupTestEnvironment()
+	baselineID := database.CreateBaseline(t)
+
+	data := `{
+		"name": "",
+		"inventory_ids": [],
+		"config": {
+			"to_time": "2021-01-01 12:00:00-04"
+		}
+	}`
+
+	w := httptest.NewRecorder()
+	path := fmt.Sprintf(`/%v`, baselineID)
+	req, _ := http.NewRequest("POST", path, bytes.NewBufferString(data))
+	core.InitRouterWithParams(BaselineUpdateHandler, 1, "POST", "/:baseline_id").ServeHTTP(w, req)
+
+	assert.Equal(t, http.StatusOK, w.Code)
+
+	var baseline models.Baseline
+	assert.Nil(t, database.Db.Model(&baseline).Where("id = ?", baselineID).Find(&baseline).Error)
+	assert.Equal(t, baseline.ID, baselineID)
+	assert.Equal(t, baseline.Name, "")
+
+	output := testUpdatedBaselineSystems(t, baselineID)
+
+	assert.Equal(t, 0, len(output.Data))
+
+	database.DeleteBaseline(t, baselineID, []string{
+		"00000000-0000-0000-0000-000000000005",
+		"00000000-0000-0000-0000-000000000006",
+	})
+}
+
+func testUpdatedBaselineSystems(t *testing.T, temporaryBaselineID int) BaselineSystemsResponse {
```

Comment:
It will be better to create checking method `database.CheckBaseline(t *testing.T, baselineID int, inventoryIDs []string, config, name string)`. This method can be used in all baseline related tests. It should test baseline has these (and only these) systems, config and name.